src/main/java/com/code_intelligence/jazzer/junit/FuzzTestExecutor.java

```diff
@@ -65,27 +72,16 @@ public static FuzzTestExecutor prepare(ExtensionContext context, String maxDurat
               fuzzTestMethod.getName(), getReadableDescriptor(fuzzTestMethod)));
     }
 
-    if (runFromCommandLine(context)) {
-      return prepareForCommandLine(context);
-    } else {
-      return prepareForTestRunner(context, maxDuration);
-    }
-  }
-
-  private static FuzzTestExecutor prepareForCommandLine(ExtensionContext context) {
-    return new CommandLineFuzzTestExecutor(context);
-  }
-
-  private static FuzzTestExecutor prepareForTestRunner(ExtensionContext context, String maxDuration)
-      throws IOException {
     Path baseDir =
         Paths.get(context.getConfigurationParameter("jazzer.internal.basedir").orElse(""))
             .toAbsolutePath();
 
-    final Class<?> fuzzTestClass = context.getRequiredTestClass();
+    List<String> originalLibFuzzerArgs = getLibFuzzerArgs(context);
+    String fakeArgv0 =
```

Comment:
Let's call this `argv0` as it may not bee fake anymore.

Reply:
done